src/gui/EffectView.cpp

```diff
@@ -134,15 +129,15 @@ void EffectView::editControls()
 {
 	if( m_subWindow )
 	{
-		if( !m_subWindow->isVisible() )
+		if( !m_controlView->isVisible() )
```

Comment:
Should hide the `parentWidget()` and let that propagate to its `widget()`, otherwise there's a bug (clicking the Controls button to hide an LV2 or LADSPA effect hides only the window content, not the frame)